module/attribute/src/Persistence/Dbal/Projector/Attribute/AttributeCreatedEventProjector.php

```diff
@@ -10,17 +10,14 @@
 namespace Ergonode\Attribute\Persistence\Dbal\Projector\Attribute;
 
 use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DBALException;
 use Ergonode\Attribute\Domain\Event\Attribute\AttributeCreatedEvent;
-use Ergonode\Core\Domain\Entity\AbstractId;
-use Ergonode\EventSourcing\Infrastructure\DomainEventInterface;
-use Ergonode\EventSourcing\Infrastructure\Exception\UnsupportedEventException;
-use Ergonode\EventSourcing\Infrastructure\Projector\DomainEventProjectorInterface;
 use JMS\Serializer\SerializerInterface;
 use Ramsey\Uuid\Uuid;
 
 /**
  */
-class AttributeCreatedEventProjector implements DomainEventProjectorInterface
+class AttributeCreatedEventProjector
```

Comment:
Separate it to different methods :D Invoke method look's like old fashion portal class :D